python/dgl/graphbolt/utils/internal.py

```diff
@@ -19,7 +19,14 @@ def _read_numpy_data(path, in_memory=True):
 def read_data(path, fmt, in_memory=True):
     """Read data from disk."""
     if fmt == "torch":
-        return _read_torch_data(path)
+        _tensor = _read_torch_data(path)
+        if not _tensor.is_contiguous():
+            Warning(
+                "The tensor read from disk is not contiguous, "
+                "so it will be copied to contiguous memory."
+            )
+            _tensor = _tensor.contiguous()
+        return _tensor
     elif fmt == "numpy":
         return _read_numpy_data(path, in_memory=in_memory)
```

Comment:
for `numpy` data, we don't need to make it `contiguous` when read? Instead, we do it when saving?